[2024-04-13 16:27:04] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:09] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:14] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:19] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:24] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:24] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 3023af312ae4d24d [::1]:41884 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:27:24] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 3023af312ae4d24d [::1]:20134 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:27:29] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:34] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:39] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:44] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:49] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:54] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 16:27:56] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 3023af312ae4d24d [::1]:47156 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:27:56] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 3023af312ae4d24d [::1]:20134 failed checkpoint posthandshake: Connected to self
[2024-04-13 16:27:59] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
